images/varnish-drupal/drupal.vcl

```diff
@@ -330,6 +329,13 @@ sub vcl_backend_response {
     set beresp.http.Expires = "" + (now + beresp.ttl);
   }
 
+  # Files larger than 10 MB get streamed.
+  if (beresp.http.Content-Length ~ "[0-9]{8,}") {
+    set beresp.do_stream = true;
+    set beresp.uncacheable = true;
+    set beresp.ttl = 0s;
```

Comment:
I think this value could result in request serialisation as per https://book.varnish-software.com/4.0/chapters/VCL_Basics.html#vcl-backend-response. Maybe it should be e.g. 120s?

Reply:
Great point, I forgot the hit-for-pass should be cached.